src/scripts/main.js

```diff
@@ -1 +1,52 @@
 'use strict';
+
+Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject('First promise was rejected');
+  }, 3000);
+
+  document.body.addEventListener(
+    'click',
+    () => resolve('First promise was resolved'),
+    clearTimeout(timeoutId),
+    { once: true },
+  );
```

Comment:
The 'addEventListener' method expects only three arguments: the event name, the event handler, and an optional options object. You are passing 'clearTimeout(timeoutId)' as a third argument, which is incorrect. The clearTimeout should be inside the click event handler.